src/issn.rs

```diff
@@ -1,16 +1,16 @@
 /// ```
 /// use library_stdnums::issn::checkdigit;
-/// assert_eq!(checkdigit("0378-5955"), 5);
+/// assert_eq!(checkdigit("0378-5955"), '5');
 /// ```
-pub fn checkdigit(issn: &str) -> u32 {
+pub fn checkdigit(issn: &str) -> char {
     let clean_string = issn.replace("-", "");
     let first_seven = clean_string.chars().take(7);
     let first_seven_digits = first_seven.filter_map(|x| x.to_digit(10));
     let multiplied = first_seven_digits.enumerate().map(|(index, digit)| digit * (8 - index as u32));
 
     let summed: u32 = multiplied.sum();
     let modulus: u32 = summed % 11;
-    return 11 as u32 - modulus;
+    return char::from_digit(11 as u32 - modulus, 11).unwrap();
```

Comment:
The `(11 as u32 - modulus, 11)`, making it base 11 rather than base 10, was Max's nutty idea that might be dumb. It might make more sense to use `(11 as u32 - modulus, 10)` and then change `10` to `X`, rather than changing `a` to `X`. 

Alternately, it might make sense to keep it as base 11, test more examples, and see if we can refactor it to use base 11 for more of the calculations (which might actually eliminate steps).